content/docs/setup/kubernetes/spec-requirements/index.md

```diff
@@ -31,3 +31,7 @@ cluster must satisfy the following requirements:
   in the metric telemetry collected by Istio.
 
 * _**Application UIDs**_: Do **not** run applications as a user with the user ID (UID) value of **1337**.
+
+* _**`NET_ADMIN` capability**:_ If pod security policies are enforced in your cluster and unless you use Istio
+CNI Plugin, your pods must have the `NET_ADMIN` capability allowed.
+See [Required Pod Capabilities](/help/ops/setup/pod-security-policies/).
```

Comment:
Need to fix this link. change to --> help/ops/setup/required-pod-capabilities/

Reply:
@tiswanso Thanks, fixed in https://github.com/istio/istio.io/pull/2891/commits/e74cd126b1072f538e923c212ec61d6c0b7ab2d1.